test/sparsedir/cholmod.jl

```diff
@@ -256,19 +257,11 @@ unsafe_store!(puint, CHOLMOD.INTLONG, 3*div(sizeof(Csize_t), 4) + 5*div(sizeof(P
 ### illegal itype
 p = ccall((:cholmod_l_allocate_sparse, :libcholmod), Ptr{CHOLMOD.C_SparseVoid},
     (Csize_t, Csize_t, Csize_t, Cint, Cint, Cint, Cint, Ptr{Void}),
-    1, 1, 1, true, true, 0, CHOLMOD.REAL, CHOLMOD.common(CHOLMOD.SuiteSparse_long))
+    1, 1, 1, true, true, 0, CHOLMOD.REAL, CHOLMOD.common())
 puint = convert(Ptr{Uint32}, p)
 unsafe_store!(puint,  5, 3*div(sizeof(Csize_t), 4) + 5*div(sizeof(Ptr{Void}), 4) + 2)
 @test_throws CHOLMOD.CHOLMODException CHOLMOD.Sparse(p)
 
-# test that Sparse(Ptr) works for SuiteSparse_long (on 64 bit systems)
-if CHOLMOD.SuiteSparse_long == Int64
-    p = ccall((:cholmod_allocate_sparse, :libcholmod), Ptr{CHOLMOD.C_SparseVoid},
-        (Csize_t, Csize_t, Csize_t, Cint, Cint, Cint, Cint, Ptr{Void}),
-        1, 1, 1, true, true, 0, CHOLMOD.REAL, CHOLMOD.common(Int32))
-    @test isa(CHOLMOD.Sparse(p), CHOLMOD.Sparse{Float64,Int32})
```

Comment:
This is covered by the other tests now the we don't support all integer sizes anymore.